packages/lesswrong/lib/collections/comments/collection.ts

```diff
@@ -5,6 +5,7 @@ import { userIsAllowedToComment } from '../users/helpers';
 import { mongoFindOne } from '../../mongoQueries';
 import { addUniversalFields, getDefaultResolvers } from '../../collectionUtils'
 import { getDefaultMutations, MutationOptions } from '../../vulcan-core/default_mutations';
+import { makeEditable } from '../../editor/make_editable';
```

Comment:
I moved this (along with other instances of `makeEditable` and `addUniversalFields`)  from `custom_fields` to `collection` rather than `schema`, since in `schema` it results in a dependency cycle. This matches the organization used in other collections, one example being Books.